scripts/automation/utilities/path.py

```diff
@@ -129,3 +138,48 @@ def filter_user_selected_modules_with_tests(user_input_modules):
                     if name in selected_modules)
     else:
         return list((name, module, test) for name, module, test in existing_modules)
+
+
+if __name__ == '__main__':
+    print('Automation utility')
+    print('All modules and their paths')
+
+    module_paths = get_all_module_paths()
+    template = '  {:' + str(max([len(m) for m, _ in module_paths]) + 2) + '}{}'
+
+    for m, p in module_paths:
+        print(template.format(m, p))
+
+    print('Total: {}\n\n'.format(len(module_paths)))
+
+    print('Core modules and their paths')
+
+    module_paths = list(get_core_modules_paths())
+    template = '  {:' + str(max([len(m) for m, _ in module_paths]) + 2) + '}{}'
+
+    for m, p in module_paths:
+        print(template.format(m, p))
+
+    print('Total: {}\n\n'.format(len(module_paths)))
+
+    print('Core modules and their paths as well as tests')
+
+    module_paths = list(get_core_modules_paths_with_tests())
+    name_len = str(max([len(m) for m, _, _ in module_paths]) + 2)
+    template = '  {:' + name_len + '}{}\n  {:' + name_len + '}{}'
+
+    for m, p, t in module_paths:
+        print(template.format(m, p, '', t))
+
+    print('Total: {}\n\n'.format(len(module_paths)))
+
+    print('Command modules and their paths as well as tests')
+
+    module_paths = list(get_command_modules_paths_with_tests())
+    name_len = str(max([len(m) for m, _, _ in module_paths]) + 2)
+    template = '  {:' + name_len + '}{}\n  {:' + name_len + '}{}'
+
+    for m, p, t in module_paths:
+        print(template.format(m, p, '', t))
```

Comment:
the code here is just for testing purpose. run `python -m automation.utilities.path` will list all the modules, their paths, and test paths. the template is used to format the columns for printing.